orderer/consensus/smartbft/chain_test.go

```diff
@@ -361,6 +360,7 @@ func TestAddAndRemoveNodeWithoutStop(t *testing.T) {
 
 	// start a network
 	networkSetupInfo := NewNetworkSetupInfo(t, channelId, dir)
+	networkSetupInfo.configInfo.leaderHeartbeatTimeout = time.Minute
```

Comment:
isn't it the default?

Reply:
Here's where I don't fully understand mock.
But I caught that nodes go to the state of view change due to heartbeat.

In this case for TestAddNodeWhileAnotherNodeIsDown test it takes 20 seconds, and for TestAddAndRemoveNodeWithoutStop test it takes 1 minute.

Otherwise they crash.

Perhaps there is something missing in the mock to function properly.